src/Feature/Account/code/Controllers/AuthenticationController.cs

```diff
@@ -54,20 +54,20 @@ public ActionResult ValidateCredentials(UserLoginModel userLogin)
         }
 
         [HttpPost]
-        [ActionName("signin")]
+        [ActionName("signIn")]
         public ActionResult SignIn(UserLoginModel userLogin, string returnUrl)
         {
             var userLoginResult = LoginUser(userLogin, out var commerceUserModel);
 
-            if (!userLoginResult || commerceUserModel == null) return Redirect("/signin");
+            if (!userLoginResult || commerceUserModel == null) return Redirect("/signIn");
 
             CompleteAuthentication(commerceUserModel);
 
-            return RedirectOnSignin(returnUrl);
+            return this.RedirectOnSignIn(returnUrl);
         }
 
         [HttpPost]
-        [ActionName("signout")]
+        [ActionName("signOut")]
```

Comment:
It's better to leave `signout`, because on FE side we have direct mention of this action in current notation.

Reply:
Fixed spelling in front end side